src/CSModule.sol

```diff
@@ -1388,6 +1447,22 @@ contract CSModule is
         }
     }
 
+    function _isEnoughStrikesToEject(
+        uint256 strikesLifetime,
+        uint256 ejectThreshold,
+        uint256[] calldata strikesData
+    ) internal view returns (bool) {
+        uint256 strikesDataLength = strikesData.length;
+        if (strikesDataLength != strikesLifetime) revert InvalidStrikesData();
```

Comment:
this check is not related, put it on the level above

Reply:
> This check is off and redundant in general. Oracle should take care of expired strikes and remove them from the tree

We should care about case when `lifetime` was changed in the contract, but new report is not happened yet